### PR TITLE
Define ARM64 version of testrunner

### PR DESCRIPTION
**Pull request by mthalman**
We need an ARM64 version of testrunner so that it can be run on ARM machines.

ARM builds of PowerShell are not available in the Debian package feeds so it is downloaded directly from the GitHub releases URL.  

Defined a multi-arch tag for testrunner to make it simpler to reference.  To avoid issues with needing to have the same timestamp value across multiple build jobs, I decided to just remove that segment of the tag.  Also had to remove the commit SHA from the tag because Image Builder doesn't support that built-in variable in shared tags since there's no build context from which to get a commit.  This uniqueness doesn't really seem necessary for how little this image gets updated.  Let me know what you think.  Otherwise, I'll need to add a custom variable to provide uniqueness.

Related to https://github.com/dotnet/docker-tools/issues/486

### CODE REVIEW

File: src/debian/stretch-slim/docker-testrunner/arm64v8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+FROM arm64v8/debian:stretch-slim
+
+# Install Docker
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+        gnupg2 \
+        libicu57 \
```
Comment (MichaelSimons) [inline src/debian/stretch-slim/docker-testrunner/arm64v8/Dockerfile:13]: Why is this required in the arm64 variant but not x64?

Comment (mthalman) [reply src/debian/stretch-slim/docker-testrunner/arm64v8/Dockerfile:13]: It is required.  But since the x64 version installs PowerShell via a package, libicu gets installed as a dependency automatically.  In the arm64 variant, we're installing PowerShell manually, so we need to explicitly include libicu.

**Comment by mthalman** [comment]
> I think the shared tag needs to be unique per change. The components in the Docker image can have breaking changes which we should be able roll-out in stages.

Added a UniqueId to the tag, just as was done for [Image Builder](https://github.com/dotnet/docker-tools/pull/553).
